CMakeLists.txt

```diff
@@ -1381,10 +1384,13 @@ if(USE_LIBSNDFILE)
 endif()
 
 if(SampleRate_FOUND)
+    if(NOT IOS)
   target_link_libraries(${CSOUNDLIB} PUBLIC SampleRate::samplerate)
+      endif()
 endif()
 
 if(NOT IOS)
+   target_link_options(${CSOUNDLIB} PUBLIC ${CSOUNDLIB_SHARED_LINK_OPTIONS})
```

Comment:
It looks like CSOUNDLIB_SHARED_LINK_OPTIONS is set to "" on line 8. Was this temporary work that should be removed now?

Reply:
no, I am using this in Custom.cmake. It is   there so it can be filled or not later
I don't know of a better way to set this up,  it's not an option.

Reply:
but I think I can make it into an option. It's for lto.

Reply:
yes, that's in there now
068238cee4203071262e9d019bb0d40a93a10a6e